src/js/tests/newsletter-subscription.test.js

```diff
@@ -91,3 +91,60 @@ test('Happy path', async () => {
     '<b>Thank you. Please check your email to confirm your subscription.</b>',
   );
 });
+
+test('IE11 manual fallback', async () => {
+  // Testing that the overall behaviour still works with IE11, which has
+  // no URLSearchParams or complete FormData implementation
+  // Store original implementation
+  const originalURLSearchParams = URLSearchParams;
+
+  // eslint-disable-next-line no-global-assign
+  URLSearchParams = jest.fn(() => {
+    throw new Error();
+  });
+
+  fetchMock.mockResponseOnce(JSON.stringify({ success: true }));
+  document.body.innerHTML = exampleFormHTML;
+  NewsletterSubscription.init(); // this is how it's invoked in index.js
+  const form = document.getElementById('newsletter-form');
+
+  populateFormData(document, {
+    email: 'test@example.com',
+    privacy: 'checked',
+  });
+  form.submit();
+
+  expect(fetch.mock.calls.length).toEqual(1);
+
+  expect(fetch.mock.calls[0][1].headers).toEqual({
+    'X-Requested-With': 'XMLHttpRequest',
+    'Content-type': 'application/x-www-form-urlencoded',
+  });
+  expect(fetch.mock.calls[0][1].method).toEqual('POST');
```

Comment:
[`.toHaveBeenCalleWith`](https://jestjs.io/docs/en/expect#tohavebeenlastcalledwitharg1-arg2-) reads better imo

Reply:
I'll meet you half way 😄 - see the diff now (in this part and earlier in the file)